### PR TITLE
Adding More Features to Stats Page

### PR DESCRIPTION
**Pull request by azanbinzahid**
I implemented only basics previously. Now adding more stats features. Not working on UI as of now.

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -84,13 +84,23 @@ public void storeMessage(Message message) {
     return messages;
   }
 
+
+
```
Comment (raufrajar) [inline src/main/java/com/google/codeu/data/Datastore.java]: nit: remove extraneous lines

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -84,13 +84,23 @@ public void storeMessage(Message message) {
...
     public int getTotalMessageCount(){
       Query query = new Query("Message");
       PreparedQuery results = datastore.prepare(query);
       return results.countEntities(FetchOptions.Builder.withLimit(1000));
     }
 
+    /** Returns the total number of users for in site. */
+    public int getTotalUserCount(){
+      Query query = new Query("User");
+      PreparedQuery results = datastore.prepare(query);
```
Comment (raufrajar) [inline src/main/java/com/google/codeu/data/Datastore.java]: nit: imo you can just move query construction to next line

Comment (azanbinzahid) [reply src/main/java/com/google/codeu/data/Datastore.java]: couldn't understand..

**Review by waahm7 — APPROVED** [review]
Great job!

**Pull request merged**